README.md

```diff
@@ -57,137 +59,68 @@ import { PrimaryButton } from 'office-ui-fabric-react/lib/Button';
+Please take a look at our [contribution guidelines](https://github.com/OfficeDev/office-ui-fabric-react/wiki/Contributing) for more info. Also read [Contribute bug fixes](https://github.com/OfficeDev/office-ui-fabric-react/wiki/Bug-Fixes) and [Contribute new component](https://github.com/OfficeDev/office-ui-fabric-react/wiki/New-Components).
 
 ## Building the repo
 
 Before you get started, **make sure you have read the [Git branch setup instructions](https://github.com/OfficeDev/office-ui-fabric-react/wiki/Setup)**
 
 To view the documentation including examples, contracts, component status, and to add functionality or fix issues locally, you can:
 
-1.  `git clone https://github.com/OfficeDev/office-ui-fabric-react.git`
-2.  `cd office-ui-fabric-react`
-3.  `npm install`
-4.  `npm start`
+1. `git clone https://github.com/OfficeDev/office-ui-fabric-react.git`
+2. `cd office-ui-fabric-react`
+3. `npm install`
+4. `npm start`
 
 This will start a demo page from the office-ui-fabric-react package folder, which will open a web browser with the example page. You can make changes to the code which will automatically build and refresh the page using live-reload.
 
-To build and run tests for all packages in the repo, you can run `npm run build` from the root.
+To build and run tests for all packages in the repo, run `npm run build` from the root.
 
-To build individual packages within the `packages/*/` folders, you can use `npm run build` in each individually. Note that because the packages are symlinked together, you must manage building dependencies in the right order, or use the `rush` tool to build to the specific package you want. (See advanced tips below.)
+To build individual packages within the `packages/*` or `apps/*` folders, `cd` to the relevant folder and run `npm run build`. Note that because the packages are symlinked together, you must manage building dependencies in the right order, or use the `rush` tool to build to the specific package you want. (See advanced tips below.)
 
-## Testing
+### Testing
 
-For testing see our [testing documentation](https://github.com/OfficeDev/office-ui-fabric-react/wiki/Testing).
+For info about testing, see our [testing documentation](https://github.com/OfficeDev/office-ui-fabric-react/wiki/Testing).
 
-## Advanced building tips
+### Advanced building tips
 
-The repo contains many packages, each which may have dependencies on each other. You can use the rush tool to build projects in the correct order, if you have it globally installed.
+The repo contains many packages, each which may have dependencies on each other. You can use Rush to build projects in the correct order, if you have it globally installed.

```

Comment:
It might be worth linking to `Rush`'s documentation website at least once if we don't already, https://rushjs.io/.